### PR TITLE
Adds 4.7.21 release notes

### PR DESCRIPTION
**Pull request by stevsmit**
No BZ.

Preview: https://deploy-preview-34813--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-7-21

To be merged 7-26. Will DM reminder.

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -2667,3 +2667,34 @@ link:https://access.redhat.com/solutions/6163712[{product-title} 4.7.19 containe
...
+
+link:https://access.redhat.com/solutions/6209792[{product-title} 4.7.21 container image list]
+
+[id="ocp-4-7-21-features"]
+==== Features
+
+[id="ocp-4-7-21-aws-sdk-version-update"]
+===== Amazon Web Services SDK update
+
+With this update, the aws-sdk-go is bumped to v1.38.35, which supports new Amazon Web Services (AWS) regions. This feature allows for new clusters to be installed in previously unknown regions without the risk of crashing the `image-registry` pod. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1977159[*BZ#1977159*].
```
Comment (bmcelvee) [inline release_notes/ocp-4-7-release-notes.adoc:2688]: Should `aws-sdk-go` be in backticks?

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -2667,3 +2667,34 @@ link:https://access.redhat.com/solutions/6163712[{product-title} 4.7.19 containe
...
+
+[id="ocp-4-7-21-bug-fixes"]
+==== Bug fixes
+
+* Previously, an incorrect Keepalived setting sometimes resulted in the VIP ending up on an incorrect system and unable to move back to the correct system. With this update, the incorrect Keepalived setting is removed so that the VIP ends up on the correct system. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1971864[*BZ#1971864*])
+
+* Previously, when a Baremetal IPI was deployed with a proxy configured, the internal machine-os image download was directed through the proxy. This corrupted the image and prevented it from being downloaded. This update fixes the Internal image traffic to `no_proxy`, so that the image download no longer uses a proxy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972291[*BZ#1972291*])
```
Comment (bmcelvee) [inline release_notes/ocp-4-7-release-notes.adoc:2695]: Should `machine-os` be in backticks?

**Comment by bmcelvee** [comment]
Couple of questions, but overall LGTM!

**Comment by vikram-redhat** [comment]
The errata are in push. Merging this.

**Pull request merged**